README.md

```diff
@@ -1,2 +1,49 @@
-# Covid19
-A website that allow the user to see covid19 cases around the world, data retrieved from a api
+## Covid19 Tracker
+This is an App that tracks the covid19 cases, death and other details from all countries, it allows the user to search for a specific country and to see the details about the covid19 status.
+## Build with
+
+Reactjs, API
+
+## Screenshot of the Project
+
+<img width="1280" alt="Screen Shot 2022-05-26 at 00 41 41" src="https://user-images.githubusercontent.com/91213045/170381532-9a353160-59ee-4991-89d4-69fa2d8029d9.png">
+
+<img width="1280" alt="Screen Shot 2022-05-26 at 00 41 51" src="https://user-images.githubusercontent.com/91213045/170381625-42e01891-00e4-497c-8ad0-649d0b6d62f0.png">
+
+
+## Deployment
+
+[Link to my project](https://app.netlify.com/sites/glittery-beijinho-f28c4c/deploys/628f329d107a5800087a3332)
+
+## Getting Started With React App
+To get a local copy of the project 
+
+run
+```
+git clone   https://github.com/YannickZahinda/Covid19
+cd Covid19
+
+npm install 
+```
+ to install dependencies:
+
+## Run into the Browser
+
+```
+npm start
+```
+To start the app in the development mode
+
+
+
+👤 **Yannick**
+
+- GitHub: [@YannickZahinda](https://github.com/YannickZahinda)
+- Twitter: [@ZahindaY](https://twitter.com/ZahindaY)
+- LinkedIn: [Yannick MULIKUZA](https://linkedin.com/in/linkedinhandle)
+
+**Acknowledgement**
+
+[NELSON Sakwa](https://www.behance.net/sakwadesignstudio)
```

Comment:
- [ ] Please structure better this sentence for acknowledgment:
- example: Credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).